trusted-committer-article.md

```diff
@@ -0,0 +1,449 @@
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. Think of TCs as a person in a community that you trust with
+important technical decisions and with mentoring contributors to get their
+contribution over the finish line. The TC role is both a demanding and a
+rewarding role to fulfill. It goes far beyond being an opinionated gatekeeper
+and it is instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, TCs represents the interests of both
+their InnerSource community and the products the community is building. They
+are concerned with the health of the community and the product. So as a TC,
+you'll have both tech oriented and community oriented responsibilities and
+we'll explore both of these dimensions in the following sections. 
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+Let's start with the _Contributor_ role. A _Contributor_ - as the name implies,
+makes contributions to an InnerSource community; code but also bug-reports,
+feature-requests or documentation. The contributor might or might not be part
+of the community. He might be sent by another team to develop a feature they
+might need. This is why we sometimes also refer to Contributors as _Guests_ or
+being part of a _Guest Team_. The _Contributor_ is responsible for "fitting in"
+and for conforming to the communities expectations and processes.
+
+The Trusted Committer is always a member of the InnerSource community, also
+sometimes referred to as the _Host Team_. In this analogy, the TC is
+responsible for both building the house and setting the house rules, to make
+sure their guests are comfortable and can work together effectively. Compared
+to the _Contributor_, a TCs have earned the responsibility to push code closer
+to production and are generally allowed to perform tasks that have a higher
+level of risk associated with them.
+
+The Product Owner is the third role in InnerSource. Whether or not there is a
+dedicated PO depends on the type of InnerSource community and the products they
+build. Especially in grass-roots type InnerSource communities like the one I
+work in, the TC also acts as a PO.  Similar to agile processes, the PO is
+responsible for defining and prioritizing requirements and stories for the
+community to implement. The PO therefore often interacts with the TC, e. g. in
+making sure that a requested or contributed feature actually belongs to the
+product. In the Host/Guest analogy, the PO would be the one of determines what
+Host and Guest work on in the communities House. Please check out our Product
+Owner Learning Path video for more detailed information.
+
+---(not added in text, yet)
```

Comment:
Wouldn't worry about adding this part at all - just let them check out the Product Owner video.